.github/workflows/cron-tests.yaml

```diff
@@ -62,3 +63,19 @@ jobs:
     - name: Memory test
       run: |
         ./scripts/docker-run-memory-test.sh
+
+  backup_and_restore:
+    name: Backup and restore
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+    env:
+      PG_VERSION: 12.6
```

Comment:
do we want a matrix here and test PG11 and PG13 too

Reply:
I will exclude PG11, because they still use separate `recovery.conf` file which is not supported by this script.